microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -568,8 +568,34 @@ def _load_drivers_for_dpdk(self) -> None:
         modprobe.load(rmda_drivers)
         modprobe.load(mellanox_drivers)
 
+    def check_dpdk_support(self) -> None:
+        # check requirements according to:
+        # https://docs.microsoft.com/en-us/azure/virtual-network/setup-dpdk
+        node = self.node
+        supported = False
+        if isinstance(node.os, Debian):
+            if isinstance(node.os, Ubuntu):
+                supported = node.os.information.version >= "18.4.0"
+            else:
+                supported = node.os.information.version >= "10.0.0"
+        elif isinstance(node.os, Redhat):
+            supported = node.os.information.version >= "7.5.0"
+        elif isinstance(node.os, Suse):
+            supported = node.os.information.version >= "15.0.0"
+        else:
+            # this OS is not supported
+            raise UnsupportedDistroException(
+                node.os, "This OS is not supported by the DPDK test suite for Azure."
+            )
+
+        if not supported:
+            raise SkippedException(
```

Comment:
This is similar, it should raise UnsupportedOsException. If this case should be skipped, the unsupported os exception should be converted to skipped exception in test case.

Reply:
Can I change UnsupportedDistroException to inherit from SkippedException instead of LisaException? It doesn't seem right that running a test on an unsupported platform results in a Fail currently.

Reply:
No, we cannot assume all UnsupportedDistroException should be skipped. In general, the tools shouldn't raise SkippedException, because the tool shouldn't decide to skip a test case. The test case should do. The tool should describe what happens with accurate message.